DAOS-17094 test: DO NOT LAND disable ABT mempool

kccain opened the pull request:
PR-repos: argobots@PR-32:2
Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-test-rpms: true
Skip-test-el-8.6--rpms: true
Skip-test-leap-15-rpms: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: test_engine_restart

Signed-off-by: Kenneth Cain <kenneth.cain@hpe.com>

### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

kccain committed:
Revert "DAOS-17094 container: increase stack size of cont_refresh_track_eph_one ULT (#15886)"

This reverts commit bce3c082cd85f636c4ca49ab7c4202b2c0c8565c.

Signed-off-by: Kenneth Cain <kenneth.cain@hpe.com>

kccain committed:
Revert "DAOS-16768 pool: larger ABT ULT stack sizes (#15822)"

This reverts commit aee0149222acbe060b01ead109e68b9faa406e5a.

Signed-off-by: Kenneth Cain <kenneth.cain@hpe.com>

kccain committed:
DAOS-17094 test: DO NOT LAND disable ABT mempool

PR-repos: argobots@PR-32:2
Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-test-rpms: true
Skip-test-el-8.6--rpms: true
Skip-test-leap-15-rpms: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: test_engine_restart

Signed-off-by: Kenneth Cain <kenneth.cain@hpe.com>

daosbuild1 commented:
Test stage Python Bandit check completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/1/execution/node/133/log

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/1/execution/node/269/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.5 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/1/execution/node/291/log

daosbuild1 commented:
Test stage Build RPM on EL 9 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/1/execution/node/384/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/1/execution/node/407/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.5 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/2/execution/node/336/log

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/2/execution/node/350/log

daosbuild1 commented:
Test stage Build RPM on EL 9 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/2/execution/node/339/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/2/execution/node/351/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.5 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/3/execution/node/355/log

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/3/execution/node/348/log

daosbuild1 commented:
Test stage Build RPM on EL 9 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/3/execution/node/351/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/3/execution/node/358/log

daosbuild1 commented:
Test stage Functional Hardware Medium MD on SSD completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/5/execution/node/911/log

kccain committed:
configure engines  with abt mprotect_strict checking

PR-repos: argobots@PR-32:2
Skip-unit-tests: true
Skip-fault-injection-test: true
Skip-test-rpms: true
Skip-test-el-8.6--rpms: true
Skip-test-leap-15-rpms: true
Skip-func-hw-test-medium-md-on-ssd: false
Test-tag: test_engine_restart

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-16004/6/execution/node/911/log
